tests/Controller/EnMarche/AdherentControllerTest.php

```diff
@@ -388,21 +386,19 @@ public function testAdherentChangePassword()
         $this->assertClientIsRedirectedTo('/parametres/mon-compte/changer-mot-de-passe', $this->client);
     }
 
-    public function testAdherentSetEmailNotifications()
+    public function testAdherentSetEmailNotifications(): void
     {
         $adherent = $this->getAdherentRepository()->findOneByEmail('carl999@example.fr');
 
-        $this->assertFalse($adherent->hasSubscribedLocalHostEmails());
```

Comment:
Why removing this tests ? They look quite important.

Reply:
Miss, delete sorry